pkgs/servers/matrix-dendrite/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub}:
+
+buildGoModule rec {
+  pname = "matrix-dendrite";
+  version = "0.3.11";
+
+  src = fetchFromGitHub {
+    owner = "matrix-org";
+    repo = "dendrite";
+    rev = "v${version}";
+    sha256 = "15xqd4yhsnnpz5n90fbny9i8lp7ki2z3fbpbd8cvsp49347rm483";
+  };
+
+  vendorSha256 = "1l1wydvi0yalas79cvhrqg563cvs57hg9rv6qnkw879r6smb2x1n";
+
+  meta = with lib; {
+    homepage = "https://matrix.org";
```

Comment:
This doesn't seem appropriate.

Reply:
If so, then synapse should also be changed https://github.com/NixOS/nixpkgs/blob/1d64c0db8524b94c73e5a3d18e3c1c21c2ac0396/pkgs/servers/matrix-synapse/default.nix#L76-L78

Reply:
Yes, I think so